content/en/opentelemetry/reference/concepts.md

```diff
@@ -14,14 +16,14 @@ further_reading:
+| Concept                      | Description                                                                                                                                                                                                                                                      |
+|------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Telemetry                    | The collection of metrics, logs, and traces that provide observations about the behaviors and performance of applications and infrastructure.                                                                                                                    |
+| [OpenTelemetry Collector][1] | A vendor-agnostic implementation for collecting and exporting telemetry data emitted by many processes. It can be configured to receive, process, and export telemetry to one or multiple destinations including storage backends and analysis tools.            |
+| [Datadog Exporter][2]        | A component that lets you forward trace, metric, and logs data from OpenTelemetry SDKs to Datadog.                                                                                                                                                               |
 | [OTLP Receiver][3]           | A component within the OpenTelemetry Collector responsible for accepting telemetry data in the OpenTelemetry Protocol (OTLP) format. OTLP is the native protocol for OpenTelemetry, designed for transferring telemetry data between the SDKs and the Collector. |
-| [Context Propagation][4]     | The mechanism used in distributed tracing to maintain trace context across different services.                                                                                                                                                                            |
-| [Semantic Conventions][5]    | OpenTelemetry makes use of a number of semantic conventions that specify names for different types of data.                                                                                                                                                               |
+| [Context Propagation][4]     | The mechanism used in distributed tracing to maintain trace context across different services.                                                                                                                                                                   |
+| [Semantic Conventions][5]    | OpenTelemetry makes use of a number of semantic conventions that specify names for different types of data.                                                                                                                                                      |
```

Comment:
This one does not read as a definition, but rather a statement of what semconvs are used for. Should we rephrase it for consistency?